src/icon/icon-registry.ts

```diff
@@ -26,13 +26,13 @@ export type IconMode = 'font' | 'svg';
     providedIn: 'root'
 })
 export class ThyIconRegistry {
-    private defaultFontSetClass = 'wt-icon';
+    private defaultFontSetClass = 'thy-icon';
```

Comment:
defaultFontSetClass only for iconFont mode, please use `wt-icon`, which is default font-icon class in https://github.com/atinc/wt-design-icons